Sources/SwiftSyntax/SyntaxNodes.swift.gyb.template

```diff
@@ -156,7 +170,11 @@ public struct ${node.name}: ${base_type}Protocol, SyntaxHashable {
     let raw = newChild?.raw ?? ${make_missing_swift_child(child)}
 %       end
     let newData = data.replacingChild(raw, at: Cursor.${child.swift_name})
+%       if node.must_uphold_invariant:
+    return ${node.name}(newData)!
```

Comment:
We definitely shouldn‘t force unwrap here. The invariant will not be satisfied if the user passes in a child that does not satisfy the invariant (e.g. an invalid integer literal). Instead, this method should return an Optional.

Reply:
Done.